src/terminal/parser/stateMachine.cpp

```diff
@@ -664,16 +684,36 @@ void StateMachine::_ActionSs3Dispatch(const wchar_t wch)
 }
 
 // Routine Description:
-// - Triggers the DcsPassThrough action to indicate that the listener should handle a DCS data string character.
+// - Triggers the DcsDispatch action to indicate that the listener should handle a control sequence.
+//   The returned handler function will be used to process the subsequent data string characters.
 // Arguments:
 // - wch - Character to dispatch.
 // Return Value:
 // - <none>
-void StateMachine::_ActionDcsPassThrough(const wchar_t wch)
+void StateMachine::_ActionDcsDispatch(const wchar_t wch)

```

Comment:
I love the cleanup in here. I'm fine with fewer states as it's less difficult to mentally juggle.

Reply:
Totally.